changelog/unreleased/issue-693

```diff
@@ -0,0 +1,12 @@
+Enhancement: Support printing snapshot size in `snapshots` command
+
+The `snapshots` command now supports printing the snapshot size for snapshots
+created using this or a future restic version. For this, the `backup` command
+now stores the backup summary statistics in the snapshot.
+
+The text output of the `snapshots` command only shows the snapshot size. The
```

Comment:
What exactly of all the discussed options is the “snapshot size” here?

Bytes added to the repo when this backup was created?

If so, fine with me, but I would _still_ **love** to also see the “amount of data backed up” (what others call restore size) for a quick double-check.

Reply:
It is `total_bytes_processed` which IMO should be the restore size.